app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -244,45 +326,45 @@ You do not need to pay an agent or any other third party to complete your applic
 
 You will not be able to use eligibility for the IRP in your visa application as proof that you have enough money to support yourself.
 
-### Payment
+### Receiving payments
 
-The international relocation payment (IRP) cannot be paid to you in advance, before you take up your teaching job. This is because eligibility requirement checks can only be completed after you’ve arrived in England.
+You will not be paid your first instalment of the IRP before you start your teaching job. This is because eligibility requirement checks can only be completed after you’ve arrived in England.
 
-In order to be paid the IRP, you must have a UK bank account in your name (this can be digital). The IRP cannot be paid into a building society account.
+If your eligibility checks are successful you'll get the second instalment of the IRP at the start of your second year of employment. 
+
+To be paid the IRP, you must have a UK bank account in your name (this can be digital). The IRP cannot be paid into a building society account.
 
 You must also have a [UK National Insurance number](https://www.gov.uk/apply-national-insurance-number). You can apply for this once you arrive in England, as long as your visa gives you the right to work and you have a confirmed job.
 
-If you are eligible, you should receive a payment from the Department for Education shortly after the end of the term of employment as a teacher in which you apply. You'll be paid as and when your eligibility requirements have been checked – we cannot guarantee an exact payment date.
+You’ll be paid the first and second instalments of the IRP after your eligibility checks are completed. We cannot guarantee an exact payment date.
```

Comment:
```suggestion
You’ll be paid the first and second instalments of the IRP after your eligibility checks are completed. We cannot guarantee an exact payment date.

To be paid the IRP, you must have a UK bank account in your name (this can be digital). The IRP cannot be paid into a building society account.

You must also have a [UK National Insurance number](https://www.gov.uk/apply-national-insurance-number). You can apply for this once you arrive in England, as long as your visa gives you the right to work and you have a confirmed job.
```